WithSecure/withsecure/isolate_device_from_network_action.py

```diff
@@ -8,13 +9,22 @@ class ActionArguments(BaseModel):
     message: str | None = None
 
 
+class RemoteOperationResponse(BaseModel):
+    multistatus: list[Any]
+    transactionId: str
```

Comment:
I suggest outsourcing this declaration in a `models.py` file and import it in this file.